### PR TITLE
SEEK-402 Split docker compose and docker

### PR DESCRIPTION
**Pull request by val06**
fixed docker deployment.
temporarily removed the docker-compose deployment, we should create a proper deployment that supports dynamic generation of the docker-compose file based on the target containers that should be tapped

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Sniffer docker will passively sniff target container traffic and move the pcaps to the remote bucket.
+Supported buckets: S3 or GCS
+
+The machine must be configured to have an IAM Role access or provide ACCESS_KEY and SECRET_KEY via the configuration to the docker  
+
+## Installation
+
+Edit the conf.env with the following values:
+
+_Bucket access parameters: (**or** use an IAM role **instead** - with AWS S3 buckets only)_
```
Comment (guyarb) [inline docker/README.md:13]: dump dir is missing

File: docker/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- S3_URL                - endpoint-url (_must be configured to https://storage.googleapis.com if the bucket is a GCS bucket_) 
+- REGION                - to provide default region for the target bucket (_optional_)
+
+- BUCKET_NAME           - name of the target bucket to send the pcaps
+- FOLDER_NAME           - name of the folder inside the bucket to store the pcaps
+- BPF_FILTER            - port number and protocol type you want to sniff (e.g: tcp port 80)
+- ROTATION_SECONDS      - time interval to rotate pcap files by the sniffer (_optional_)
+- MAX_FILES             - max files to store locally before moving to the bucket (_optional_)
+- MAX_FILE_SIZE         - max size of the single pcap file (_optional_)
+- ROTATION_SECONDS      - time interval to rotate pcap files by the sniffer (_optional_)
```
Comment (guyarb) [inline docker/README.md:25]: duplicated with line 22

File: docker/README.md (hunk 1 above)
Comment (guyarb) [reply docker/README.md:13]: prefix is missing

**Review by guyarb — CHANGES_REQUESTED** [review]

File: docker/README.md (hunk 1 above)
Comment (val06) [reply docker/README.md:13]: fixed

File: docker/README.md (hunk 2 above)
Comment (val06) [reply docker/README.md]: removed

**Review by guyarb — APPROVED** [review]

**Pull request merged**